inc/views/base/error/error.twig

```diff
@@ -0,0 +1,16 @@
+{% extends 'layouts/master.twig' %}
+
+{% block head %}
+    <title>{% if title %}{{ title }}{% else %}{{ mybb.settings.bbname }}{% endif %}</title>
+{% endblock head %}
+
+{% block body %}
+<table border="0" cellspacing="{{ theme.borderwidth }}" cellpadding="{{ theme.tablespace }}" class="tborder">
+    <tr>
+        <td class="thead"><span class="smalltext"><strong>{% if title %}{{ title }}{% else %}{{ mybb.settings.bbname }}{% endif %}</strong></span></td>
```

Comment:
Personally I'd prefer to have this spread across multiple lines, like:

```twig
{% if title %}
    {{ title }}
{% else %}
    {{ mybb.settings.bbname }}
{% endif %}
```

I don't mind things all in one line if it's not too long, but this is quite a long line to read.